packages/components/Organisms/SiteHeader/search.vue

```diff
@@ -63,6 +66,22 @@ export default {
     padding: $rpl-site-header-search-padding;
     box-sizing: content-box;
 
+    .rpl-menu__quickexit {
```

Comment:
Are you able to restructure this and move it below the breakpoints?

breakpoint styles show flow down..
mobile
media xs
media s
media m
media l
... and so on.
Also, is worth the quick exit having it's own class for this case? Just so the naming and styles are scoped to this component?